src/extension.ts

```diff
@@ -30,6 +30,9 @@ import config from './config';
 import handleNewCljFiles from './fileHandler';
 import lsp from './lsp';
 import * as snippets from './custom-snippets';
+import * as docMirror from './doc-mirror'
+import { testDataDir } from './extension-test/integration/suite/util';
```

Comment:
I think maybe this was imported accidentally or left after some temporary test? Probably can remove.